staff/mgarciarojas/Javascript exercises/Hexadecimal.js

```diff
@@ -0,0 +1,20 @@
+//Hexadecimal
+//Writes a function that convert a hexadecimal color, "blue" for example
+// "#0000FF" in its RGB representation rgb(0,0,255)". 
+//Give the function the name getRGB() and test it with this code
+
+function getRGB(hexaColor){
+   var hexadecimal=hexaColor;
+   var rgb="rgb (";
+   for (var i =1; i<7; i=i+2){
```

Comment:
Would the use of [`substring`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring) simplify this logic? Have a look at it

Reply:
done

Reply:
done